### PR TITLE
Window size setting.

### PR DESCRIPTION
**Pull request by RubenNL**

### CODE REVIEW

**Comment by MightyTitan** [comment]
The scaling does not always work properly. Not all the elements are scaled.  This particular element was working and then just stopped working.
Image 001

**Comment by briangann** [comment]
let me test this out, i have seen this also on some dashboards.

File: pkg/cmd/grafana-kiosk/main.go [hunk 1]
```diff
@@ -44,6 +46,8 @@ func ProcessArgs(cfg interface{}) Args {
 	f.StringVar(&a.Mode, "kiosk-mode", "full", "Kiosk Display Mode [full|tv|disabled]\nfull = No TOPNAV and No SIDEBAR\ntv = No SIDEBAR\ndisabled = omit option\n")
 	f.StringVar(&a.URL, "URL", "https://play.grafana.org", "URL to Grafana server")
 	f.StringVar(&a.WindowPosition, "window-position", "0,0", "Top Left Position of Kiosk")
+	f.StringVar(&a.WindowPosition, "window-size", "1920,1080", "Size of Kiosk in pixels")
```
Comment (briangann) [inline pkg/cmd/grafana-kiosk/main.go:49]: this variable should be `a.WindowSize` or it will overwrite the position setting and not set the size

**Comment by briangann** [comment]
this has been implemented/merged as part of  PR #102

**Comment by koliyo** [comment]
The zoom scaling factor, chrome flag `force-device-scale-factor`, is not part of the #102 PR

**Comment by nils-vixel** [comment]
New PR here for scaling factor: https://github.com/grafana/grafana-kiosk/pull/109

**Pull request closed without merging**